ag-grid-webcomponent/date_range_filter.js

```diff
@@ -0,0 +1,105 @@
+class DateRangeFilter {
```

Comment:
What does this filter even do? It is not clear to me. Whether a selected day in a list of start and end dates?